lib/hammer_cli_katello/content_view_version.rb

```diff
@@ -287,18 +287,31 @@ class ExportCommand < HammerCLIForeman::Command
 
       def execute
         cvv = show(:content_view_versions, 'id' => options['option_id'])
-        repositories = cvv['repositories'].collect do |repo|
-          show(:repositories, 'id' => repo['id'], :full_result => true)
-        end
+        cv = show(:content_views, 'id' => cvv['content_view_id'])
 
-        check_repo_download_policy(repositories)
+        composite = cv["composite"]
+        export_json_options = { :cvv => cvv }
 
-        repositories.each do |repo|
-          repo['packages'] = index(:packages, 'repository_id' => repo['id'], :full_result => true)
-          repo['errata'] = index(:errata, 'repository_id' => repo['id'], :full_result => true)
+        if composite
+          components = cv['components']
+          export_json_options[:component_cvvs] = \
+            components.collect { |component| component['name'] }
```

Comment:
I'd rather see this as:

```
export_json_optionsp[:component_cvvs] = components.collect do | component|
  component['name']
end
```

if having to split over multiple lines due to line length.